package.json

```diff
@@ -58,6 +58,8 @@
     "mz": "2.6.0",
     "node-firefox-connect": "1.2.0",
     "regenerator-runtime": "0.10.0",
+    "parse-json": "2.2.0",
+    "regenerator-runtime": "0.9.5",
```

Comment:
"regenerator-runtime" is included twice (it is already on line 60), we should remove the "0.9.5" version and move "parse-json" to keep the dependencies ordered by name.